src/tests/spec/cookies/cookies-banner.spec.js

```diff
@@ -68,23 +94,33 @@ describe('Component: Cookie banner', function() {
 function renderComponent() {
   const formHTML =
     '<div class="cookies-banner">' +
-    '<div class="container">' +
-    '<div class="grid grid--flex grid--gutterless grid--vertical-center grid--no-wrap@s">' +
-    '<div class="grid__col grid__col--flex col-auto u-flex-shrink@s">' +
-    '<p class="cookies-banner__desc u-mr-s">CENSUS.GOV.UK uses cookies to make the site simpler.</p>' +
-    '</div>' +
-    '<div class="grid__col grid__col--flex col-auto">' +
-    '<button type="button" class="btn btn--small js-accept-cookies">' +
+    '<div class="container cookies-banner__primary">' +
```

Comment:
Would be be better if this was indented

Reply:
That wouldn't be the correct way to do it